corpus/accounts/admin.py

```diff
@@ -12,7 +12,7 @@ class CorpusUserAdmin(UserAdmin):
         (None, {"fields": ("email", "password")}),
         (
             "Personal Info",
-            {"fields": ("first_name", "last_name", "phone_no", "gender")},
+            {"fields": ("first_name", "last_name", "phone_no", "gender", "profile_pic")},
```

Comment:
Ideally shouldn't be part of this diff since it is something separate and not pertaining to the diff.....

But I'll allow it once xD